packages/react/src/components/textInput/TextInput.tsx

```diff
@@ -124,6 +124,12 @@ export type TextInputProps = MergeAndOverrideProps<
      * Button aria-label
      */
     buttonAriaLabel?: string;
+    /**
+     * The cross-origin attribute which is used to request data from a server that is not the same as the current page
+     * TODO: This is here just to fix an issue with react/types in the TextInput component. It should be removed once the issue is fixed.
+     * https://github.com/creativetimofficial/material-tailwind/issues/427
+     */
+    crossOrigin?: 'anonymous' | 'use-credentials';
   }
```

Comment:
Approved, but just a comment: Should we force this to be undefined and only undefined. If someone sets this to any value, console will show error like: "React does not recognise the "crossOrigin" property for input."